recipes/qt-wayland/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set version = "5.15.8" %}
+package:
+  name: qt-wayland
+  version: {{ version }}
+
+source:
+  url: https://download.qt.io/official_releases/qt/{{ version.rpartition('.')[0] }}/{{ version }}/submodules/qtwayland-everywhere-opensource-src-{{ version }}.tar.xz
+  sha256: fa0e7bd40f5da2d30fbdb51a436c5776950d6bd070b61a8d62d85cb537934529
+
+build:
+  skip: true   # [not linux]
+  number: 0
+  detect_binary_files_with_prefix: true
+  run_exports:
+    - {{ pin_subpackage('qt-wayland', max_pin='x.x') }}
+
+requirements:
+  build:
+    - perl
+    - make
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - qt-main                            # [build_platform != target_platform]
+    - sysroot_linux-64 2.17              # [linux]
+    - {{ cdt('mesa-libgl-devel') }}      # [linux]
+    - {{ cdt('mesa-libegl') }}           # [linux]
+    - {{ cdt('mesa-libegl-devel') }}     # [linux]
+    - {{ cdt('mesa-dri-drivers') }}      # [linux]
+    - {{ cdt('libdrm-devel') }}          # [linux]
+    - {{ cdt('libglvnd-glx') }}          # [linux]
+    - {{ cdt('libglvnd-egl') }}          # [linux]
```

Comment:
* I guess this is necessary to build Qt Wayland for more drivers, right?
* Also, could you update the PR title to remove the `no cdts` part?

Reply:
The CDT part referred to the slew of non-opengl CDTs.

Reply:
Is there a way for me to search for the available flags for a qmake project? I would like to affirm something like:
```
-egl
```

in the build process to ensure that future cleanup do not remove such a critical feature.

https://github.com/conda-forge/qt-main-feedstock/pull/122/files#diff-d7075654874cb08007a21aaab3ecd4b3453a9087e7505d034d548b8938b599bcR126

Reply:
> Is there a way for me to search for the available flags for a qmake project?

No idea, sorry.